src/pg/sql/42_observatory_exploration.sql

```diff
@@ -0,0 +1,30 @@
+
+-- return a table that contains a string match based on input
+-- TODO: implement search for timespan
+
+CREATE OR REPLACE FUNCTION OBS_SearchTables(
```

Comment:
Since this is primarily returning `tablename`, which is for the most part internal facing, I think this should be private and preceded by a `_`.